### DISCUSSION TITLE
Please help me to collect nice Book:^)

### DISCUSSION THREAD

**Discussion by yuka1369** [NAIST-SE/AttractingContributors#17] [General]
No more books, no more life.I want to know your favorite books. I would be happy if you could write bellow :)
https://github.com/yuka1369/AttractingContributors-Book.git

**Comment by sergiodl-naist** [comment]
Added the one that I keep revisiting a lot!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Reply by yuka1369** [reply]
コメントに気づくのが遅くなって申し訳ないです。コメントありがとうございます！
プルリクでやり取り致しました通り、コミットしました＾＾

感謝を込めてsergiodl-naistさんのリストにプルリクエストを送付させて頂きました。

**Comment by yokoro13** [comment]
I added!
Please help me!
https://github.com/yokoro13/AttractingContributors-VideoGames

**Reply by yuka1369** [reply]
コメントに気づくのが遅くなって申し訳ないです。コメントありがとうございます！
プルリクでやり取り致しました通り、コミットしました＾＾

感謝を込めてyokoro13さんのリストにプルリクエストを送付させて頂きました。

**Comment by iikyara** [comment]
I added Biblia Koshodo no Jiken Techo!
Please help me too!
https://github.com/iikyara/AttractingContributors-Gifts

**Reply by yuka1369** [reply]
コメントに気づくのが遅くなって申し訳ないです。コメントありがとうございます！
プルリクでやり取り致しました通り、コミットしました＾＾

感謝をこめてiikyaraさんのリストにプルリクエストを送付させて頂きました。

**Comment by zhangyongxin1212** [comment]
Added, plz help me too
https://github.com/zhangyongxin1212/AttractingContributors-Game

**Reply by yuka1369** [reply]
コメントに気づくのが遅くなって申し訳ないです。コメントありがとうございます！
プルリクでやり取り致しました通り、コミットしました＾＾

感謝をこめてzhangyongxin1212さんのリストにプルリクエストを送付させて頂きました。
